keras2onnx/ke2onnx/activation.py

```diff
@@ -4,7 +4,7 @@
 # license information.
 ###############################################################################
 import numpy as np
-import tensorflow as tf
+from keras2onnx.proto.tfcompat import tensorflow as tf
```

Comment:
good finding, you could be add both tensorflow.compat.v1.nn and tensorflow.nn

Reply:
good, updated.